samples/unit_testing/subscription-manager/unit_tests/test_createsubscription_suborchestrator.py

```diff
@@ -0,0 +1,176 @@
+import datetime
+import azure.durable_functions as df
+import azure.functions as func
+from datetime import timedelta
+from unittest import main
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+from ..Subscription.subscription import SubscriptionProperties, SubscriptionResponse
+from ..CreateSubscriptionSubOrchestrator import orchestrator_fn
+
+#######################################################################
+#### Call Activity Mocks ##############################################
+#######################################################################
+def call_activity_create_subscription(activityName: str, payload):
+    assert activityName == "CreateSubscription"
+    mock.call_activity.side_effect = call_activity_mock_status_check_succeeded
+
+def call_activity_mock_status_check_accepted(activityName: str, payload):
+    assert activityName == "StatusCheck"
+
+def call_activity_mock_status_check_succeeded(activityName: str, payload):
+    assert activityName == "StatusCheck"
+
+def call_activity_mock_status_check_notfound(activityName:str, payload):
+    mock.call_activity.side_effect = call_activity_create_subscription
+    assert activityName == "StatusCheck"
+
+def call_activity_mock_status_check_error(activityName:str, payload):
+    assert activityName == "StatusCheck"
```

Comment:
if most of these are do the same, I think it would help simplify things if we just had one method for them all :)

Reply:
Yes - I just thought if anyone wants to customize a very specific activity, they could and it helps makes it readable